src/main/java/com/puppycrawl/tools/checkstyle/JavaParser.java

```diff
@@ -236,4 +239,83 @@ private static DetailAST createSlCommentNode(Token token) {
         return slComment;
     }
 
+    /**
+     * Create block comment from token.
+     * @param token
+     *        Token object.
+     * @return DetailAST with BLOCK_COMMENT type.
+     */
+    public static DetailAST createBlockCommentNode(Token token) {
```

Comment:
Method was moved to further restrict where antlr can be used. `Token` is an antlr class.

Reply:
I moved these to the other issue just to simplify fixing the CI.